src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -444,40 +452,42 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
   }
 
   getUpdatedName (originalName: string): string {
-    if (this.isBcUlcCompany) {
```

Comment:
I think this is another bug as above.

Instead of checking the current entity type using the store getter, this should be checking the selected entity type using the computed.